src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -21,6 +21,10 @@ if not fs.exists( sPath ) and not string.find( sPath, "%." ) then
     end
 end
 
+if multishell and settings.get( "edit.multishell_title" ) == true then
```

Comment:
There isn't any need for `== true`. I know you prefer it, but let's keep it consistent with the rest of the code :).